Synchronized Spanish README with English README (#2065)
alan-turing-institute/the-turing-way: README-translated/README-Spanish.md

| **Compilación** | [![Netlify build](https://github.com/alan-turing-institute/the-turing-way/workflows/CI/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ACI+branch%3Amaster) [![Latin Phrases](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Latin%20Phrases/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Latin+Phrases%22+branch%3Amaster) [![TestFileSizes](https://github.com/alan-turing-institute/the-turing-way/workflows/TestFileSizes/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ATestFileSizes+branch%3Amaster) [![Check for Lorem Ipsums](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Lorem%20Ipsums/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Lorem+Ipsums%22+branch%3Amaster) |
| **Cita** |  [![Cite this using DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3233853.svg)](https://doi.org/10.5281/zenodo.3233853) |

> *Bienvenidos al repositorio de Github del proyecto The Turing Way. Aquí es donde todos los componentes del proyecto se desarrollan, se revisan y se mantienen.*

Reviewer: ```suggestion
*Bienvenidos al repositorio de Github del proyecto The Turing Way. Aquí es donde se desarrollan, se revisan y se mantienen todos los componentes del proyecto.*
```

> _The Turing Way_ es una guía asertiva para la ciencia de datos reproducible. 

Reviewer: Asertiva??
Que quiere decir?

Where's the original text where this was extracted from? I can see the English readme, but these paragraphs aren't contained there?

> Actualmente estás en el repositorio de GitHub del proyecto donde se encuentran todos los fragmentos que componen la guía y donde ocurre el proceso de escritura/construcción de la misma.

Reviewer: ```suggestion
```
This is just said a bit before, seems like a repetition

> Nuestro objetivo es proporcionar toda la información que las personas que hacen investigación necesitan al inicio de sus proyectos para garantizar que sean fáciles de reproducir al final.

Reviewer: ```suggestion
Nuestro objetivo es proporcionar toda la información que los investigadores y científicos de datos en la academia, la industria y el sector público necesitan al inicio de sus proyectos para asegurar que sean fáciles de reproducir al final.
```
He completado un poco con lo que aparece en la version inglesa

[![El proyecto The Turing Way es un libro, una comunidad, un proyecto de código abierto y una cultura de colaboración. Esto se muestra en cuatro ilustraciones, la primera mostrando el libro de Turing Way, la segunda cómo la comunidad puede crecer, la tercera mostrando dos personas colaborando en una pull request, y la última mostrando el balance donde la reproducibilidad es valorada en mayor medida que el número de documentos publicados](../book/website/figures/README_imgs/README_turingway.png)](https://docs.google.com/presentation/d/13Nm8LcRW87ffxEugGEs5j6HKQEvhiuH8b7MfIdX7MpI/edit#slide=id.p1).

*The Turing Way es un libro, una comunidad y una colaboración global.*

> Todas las partes interesadas, incluyendo estudiantes, investigadores, ingenieros de software, líderes de proyectos y equipos fundadores, están animados a usar The Turing Way para entender sus roles y sus responsabilidades en la reproducibilidad de la ciencia de datos.

Reviewer 2: ```suggestion
Todas las partes interesadas, incluyendo estudiantes, investigadores, ingenieros de software, líderes de proyectos y equipos fundadores, están invitados a usar The Turing Way para entender sus roles y sus responsabilidades en la reproducibilidad de la ciencia de datos.
```

> Todas las partes interesadas, incluyendo estudiantes, investigadores, ingenieros de software, líderes de proyectos y equipos fundadores, están animados a usar The Turing Way para entender sus roles y sus responsabilidades en la reproducibilidad de la ciencia de datos.

Reviewer: ```suggestion
Todas las partes interesadas, incluyendo estudiantes, investigadores, ingenieros de software, líderes de proyectos y equipos fundadores, están invitadas a usar The Turing Way para entender sus roles y sus responsabilidades en la reproducibilidad de la ciencia de datos.
```

Hace referencia a las "partes interesadas", así que sería en femenino "interesadas"

Author: Solved in 4a14427d83d47024a7d422e652c3ebe0664e216e

> Todas las partes interesadas, incluyendo estudiantes, investigadores, ingenieros de software, líderes de proyectos y equipos fundadores, están invitados a usar The Turing Way para entender sus roles y sus responsabilidades en la reproducibilidad de la ciencia de datos.

Reviewer: Shall we use inclusive language?

Reviewer 3: ```suggestion
Todas las partes interesadas, incluyendo estudiantes, investigadores/as, ingenieros/as de software, líderes de proyectos y equipos fundadores, están invitados/as a usar The Turing Way para entender sus roles y sus responsabilidades en la reproducibilidad de la ciencia de datos.
```

> Puede leer el libro [online](https://github.com/alan-turing-institute/the-turing-way), contribuir al proyecto tal como se describe en nuestras [pautas de contribución](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) y reutilizar todos los materiales ([ver la Licencia](https://github.com/alan-turing-institute/the-turing-way/blob/master/LICENSE.md)).

Reviewer: ```suggestion
Puede leer el libro [online](https://github.com/alan-turing-institute/the-turing-way), contribuir al proyecto tal como se describe en nuestras [guías para contribuir al proyecto](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) y reutilizar todos los materiales ([ver la Licencia](https://github.com/alan-turing-institute/the-turing-way/blob/master/LICENSE.md)).
```

[![Captura de pantalla del libro en línea de Turing Way. También muestra una de las ilustraciónes de Turing Way al comienzo del libro. En esta ilustración, podemos ver una carretera o camino con tiendas para las diferentes habilidades de ciencia de datos. La gente puede entrar y salir con su lista de la compra y escoger lo que necesiten.](../book/website/figures/README_imgs/README_book.png)](https://the-turing-way.netlify.app/welcome.html)

*Captura de pantalla del libro en línea de The Turing Way ([usar esta imagen en una presentación](https://drive.google.com/file/d/1wJR664YECSc8b_RSHeyVjDlHs-Ls9lkc/view?usp=sharing))*

> Empezó en 2019 como una pequeña guía de ciencia de datos, y desde entonces The Turing Way se ha expandido en una serie de guías de Investigación Reproducible, Diseño de Proyectos, Comunicación, Colaboración e Investigación Ética.

Reviewer: ```suggestion
The Turing Way comenzó en 2019 como una pequeña guía de ciencia de datos, y desde entonces se ha expandido hasta formar manual mucho más completo formado por múltiples guías que abarcan varios temas: Investigación Reproducible, Diseño de Proyectos, Comunicación, Colaboración e Investigación Ética.
```
No es literalmente lo que pone en la versión inglesa, he añadido algunas palabras nuevas, pero sino la traducción sonaba regular

> Cada guía ofrece capítulos sobre una variedad de temas que cubren las mejores prácticas, guía y recomendaciones.

Reviewer: ```suggestion
Cada guía ofrece capítulos sobre una variedad de temas que cubren buenas prácticas y recomendaciones.
```
He quitado "guia" porque está repetido, "cada guía" ofrece "guía".

> Estos capítulos han sido co-escritos por colaboradores que son estudiantes, investigadores, educadores, líderes comunitarios, políticos y profesionales de diversos orígenes, experiencias vividas y conocimiento del dominio.

Reviewer: Shall we use inclusive language?

Reviewer 3: ```suggestion
Estos capítulos han sido co-escritos por personas que colaboran y que son estudiantes, investigadores/as, docentes, líderes comunitarios, políticos/as y profesionales de diversos orígenes, experiencias vividas y conocimiento del dominio.
```

***Nuestro objetivo es hacer que la reproducibilidad sea "demasiado fácil como para no hacerla".***

**Tabla de contenidos:**

- [Acerca del proyecto](#acerca-del-proyecto)
- [El Equipo](#el-equipo)
- [Colaborar](#colaborar)
- [Citando _The Turing Way_](#citando-the-turing-way)
- [Contáctanos](#contáctanos)
- [Colaboradores](#colaboradores)

🎧 Si prefieres una introducción de audio al proyecto (en inglés) Rachel, una colaboradora del proyecto presentó en el [Open Science Fair 2019](https://www.opensciencefair.eu/) en Oporto y su presentación fue grabada por el [Podcast Orion](https://orionopenscience.podbean.com/e/the-fair-is-in-town-figshare-the-turing-way-and-open-science-quest-at-the-osfair2019/).
la descripción general del _The Turing Way_ comienza en el minuto 5:13. 

### Acerca del Proyecto

La investigación reproducible es necesaria para garantizar que se pueda confiar en el trabajo científico.

> Quienes financian y editan trabajos científicos están comenzando a exigir que las publicaciones incluyan acceso a los datos subyacentes y al código de análisis.

Reviewer: ```suggestion
Quienes financian y editan trabajos científicos están comenzando a exigir que las publicaciones incluyan acceso a los datos utilizados y al código con el que se ha realizado el análisis.
```

### El equipo

_The Turing Way_ es un proyecto de colaboración abierta e impulsado por la comunidad.
Todos los que colaboran en este libro, independientemente de lo pequeñas o grande que sean sus contribuciones, están reconocidos en este proyecto como colaboradores y miembros de la comunidad. 
Colaboradores de larga duración del proyecto están considerados parte del núcleo de grupos de colaboradores que asumen roles de liderazgo.

El proyecto está coordinado por la líder investigadora **Kirstie Whitaker** (fundadora) y **Malvika Sharan** (desarrolladora de la comunidad) y alojado en [The Alan Turing Institute](https://www.turing.ac.uk/).

Puedes leer [el proceso de reconocimiento de _The Turing Way_ ](https://the-turing-way.netlify.app/community-handbook/acknowledgement/acknowledgement-members.html#community-members-contributors-and-co-authors) y [Récord de Colaboradores](https://the-turing-way.netlify.app/afterword/contributors-record.html) para aprender cómo reconocemos tu trabajo y como nuestros colaboradores son destacados en el proyecto.
Por favor visita la [Tabla de Colaboradores](#colaboradores) para ver el perfil de Github de todos nuestros colaboradores.

> ### Colaborar

Reviewer 2: ```suggestion
### Contribuir
```

> Puedes leer [el proceso de reconocimiento de _The Turing Way_ ](https://the-turing-way.netlify.app/community-handbook/acknowledgement/acknowledgement-members.html#community-members-contributors-and-co-authors) y [Récord de Colaboradores](https://the-turing-way.netlify.app/afterword/contributors-record.html) para aprender cómo reconocemos tu trabajo y como nuestros colaboradores son destacados en el proyecto.

Reviewer: ```suggestion
Puedes leer cómo se realiza [el proceso de reconocimiento como parte de _The Turing Way_ ](https://the-turing-way.netlify.app/community-handbook/acknowledgement/acknowledgement-members.html#community-members-contributors-and-co-authors) y el [Registro de Colaboradores](https://the-turing-way.netlify.app/afterword/contributors-record.html) para conocer cómo se realiza el reconocimiento del trabajo y nuestros colaboradores destacados.
```

> Por favor visita la [Tabla de Colaboradores](#colaboradores) para ver el perfil de Github de todos nuestros colaboradores.

Reviewer: ```suggestion
Por favor visita el [Panel de Colaboradores](#colaboradores) para ver el perfil de Github de todos nuestros colaboradores.
```

> :construction: Este repositorio es siempre un trabajo en progreso y **todas las personas** están invitadas a ayudarnos a construir algo que sea útil para muchas personas. :construction:

Reviewer: ```suggestion
:construction: Este repositorio es un trabajo en progreso contínuo y **todas las personas** están invitadas a ayudarnos a construir algo que sea útil para los demás. :construction:
```

> Se espera de todos los que deseen unirse al proyecto que sigan nuestro [código de conducta](../CODE_OF_CONDUCT.md) y que visiten nuestras [pautas de contribución](../CONTRIBUTING.md) para más información sobre cómo empezar.

Reviewer: ```suggestion
Se espera que todas las personas que deseen unirse al proyecto sigan nuestro [código de conducta](../CODE_OF_CONDUCT.md) y que visiten nuestras [guías para contribuir al proyecto](../CONTRIBUTING.md) para más información sobre cómo empezar.
```

> Queremos conocer dónde están nuestros colaboradores.

Reviewer: ```suggestion
Queremos conocer a nuestros colaboradores allá donde estén.
```

También puedes compartir la URL legible por humanos a una página del libro, por ejemplo, <https://the-turing-way.netlify.com/reproducibility/03/definitions.html>, pero ten en cuenta que el proyecto está en desarrollo y, por lo tanto, estos vínculos pueden cambiar con el tiempo.
Es posible que desees incluir un enlace de archivo web como: <https://web.archive.org/web/20191030093753/https://the-turing-way.netlify.com/reproducibility/03/definitions.html> para asegurarte de no terminar con enlaces rotos en todas partes.

Realmente agradecemos cualquier referencia que hagas al proyecto _The Turing Way_ y esperamos que sea de utilidad.
Si tienes alguna pregunta por favor [contáctanos](#get-in-touch).

### Citando las ilustraciones de _The Turing Way_
<p align="center">
  <img src="../book/website/figures/evolution-open-research.jpg" alt="Esto es un ejemplo de una de las ilustraciones de The Turing Way. Intenta mostrar la evolución frente a la era de la ciencia abierta" width="600">
</p>

> Las ilustraciones del _The Turing Way_ son creadas por artistas de [Scriberia](https://www.scriberia.co.uk/) como parte de [_The Turing Way_ book dashes](https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash) en Manchester el 17 de mayo de 2019, en Londres el 28 de mayo de 2019 y el 21 de febrero de 2020.

Reviewer: ```suggestion
Las ilustraciones del _The Turing Way_ se han creado por artistas de [Scriberia](https://www.scriberia.co.uk/) como parte de [_The Turing Way_ book dashes](https://github.com/alan-turing-institute/the-turing-way/tree/master/workshops/book-dash) en Manchester el 17 de mayo de 2019, en Londres el 28 de mayo de 2019 y el 21 de febrero de 2020.
```

<a href="https://the-turing-way.netlify.app/welcome.html"><img src="../book/website/figures/logo/logo.jpg?raw=true)" width="180" align="Right" /></a>

# _The Turing Way_

>_Este archivo README-Spanish.md también está disponible en alemán ([README-German.md](README-German.md)), coreano ([README-Korean](README-Korean.md)), francés ([README-French.md](README-French.md)), indonesio ([README-Indonesian](README-Indonesian.md)), inglés ([README](../README.md)), italiano ([README-Italian](README-Italian.md)), holandés ([README-Dutch](README-Dutch.md)) y portugués ([README-Portuguese](README-Portuguese.md)) (ordenados alfabéticamente)._

*Colaboradores totales:*

> <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
> [![All Contributors](https://img.shields.io/badge/all_contributors-277-orange.svg)](#contributors)
> <!-- ALL-CONTRIBUTORS-BADGE:END -->

Reviewer 4: ```suggestion
```

Reviewer 4: Removing contributors table as it only gets updated on one README (landing page).
